app/components/primer/beta/page_layout.rb

```diff
@@ -0,0 +1,504 @@
+          system_arguments[:classes]
+        )
+      end
+
+      def render?
+        content_region.present? && pane_region.present?
+      end
+
+      # The layout's content.
+      class Content < Primer::Component
+        status :beta
+
+        WIDTH_DEFAULT = :fluid
+        WIDTH_OPTIONS = [WIDTH_DEFAULT, :md, :lg, :xl].freeze
+
+        TAG_DEFAULT = :div
+        TAG_OPTIONS = [TAG_DEFAULT, :main].freeze
+
+        # @param width [Symbol] <%= one_of(Primer::Beta::PageLayout::Content::WIDTH_OPTIONS) %>
+        # @param tag [Symbol] <%= one_of(Primer::Beta::PageLayout::Content::TAG_OPTIONS) %>
+        # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+        def initialize(tag: TAG_DEFAULT, width: WIDTH_DEFAULT, **system_arguments)
+          @width = fetch_or_fallback(WIDTH_OPTIONS, width, WIDTH_DEFAULT)
+
+          @system_arguments = system_arguments
+          @system_arguments[:tag] = fetch_or_fallback(TAG_OPTIONS, tag, TAG_DEFAULT)
+          @system_arguments[:classes] = class_names(
+            "PageLayout-region",
+            "PageLayout-content",
+            system_arguments[:classes]
+          )
+        end
+
+        def call
+          render(Primer::BaseComponent.new(**@system_arguments)) do
+            if @width == :fluid
+              content
+            else
+              render(Primer::BaseComponent.new(tag: :div, classes: "PageLayout-content-centered-#{@width}")) do
+                render(Primer::BaseComponent.new(tag: :div, container: @width)) do
+                  content
+                end
+              end
+            end
+          end
+        end
+      end
+
+      # The layout's pane content. This is a secondary, smaller region that is paired with the `Content` region.
+      class Pane < Primer::Component
+        status :beta
+
+        POSITION_DEFAULT = :start
+        POSITION_MAPPINGS = {
+          POSITION_DEFAULT => "PageLayout--panePos-start",
+          :end => "PageLayout--panePos-end"
+        }.freeze
+        POSITION_OPTIONS = POSITION_MAPPINGS.keys.freeze
+
+        WIDTH_DEFAULT = :default
+        WIDTH_MAPPINGS = {
+          WIDTH_DEFAULT => "",
+          :narrow => "PageLayout--paneWidth-narrow",
+          :wide => "PageLayout--paneWidth-wide"
+        }.freeze
+        WIDTH_OPTIONS = WIDTH_MAPPINGS.keys.freeze
+
+        POSITION_NARROW_DEFAULT = :inherit
+        POSITION_NARROW_MAPPINGS = {
+          POSITION_NARROW_DEFAULT => "",
+          :start => "PageLayout--responsive-panePos-start",
+          :end => "PageLayout--responsive-panePos-end"
+        }.freeze
+        POSITION_NARROW_OPTIONS = POSITION_NARROW_MAPPINGS.keys.freeze
+
+        DIVIDER_DEFAULT = :start
+        DIVIDER_MAPPINGS = {
+          DIVIDER_DEFAULT => "PageLayout--panePos-start",
+          :end => "PageLayout--panePos-start"
+        }.freeze
```

Comment:
In the [CSS Storybook], there aren't "start" and "end" options for `divider`. Pane either has a divider or it doesn't